backend/packages/Upgrade/src/database/migrations/1703000000000-dropExperimentUserFKFromExposures.ts

```diff
@@ -0,0 +1,17 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class DropExperimentUserFKFromExposures1703000000000 implements MigrationInterface {
```

Comment:
Is there a reason we want this migration to seem to have been created on December 19, 2023 (1703000000000)? That puts it in a different place in the migrations list than it would be if it had been generated by running `npm run migration:generate --name=dropExperimentUserFKFromExposures`.  In particular, it breaks running migrations after a db drop because it attempts to alter a table that doesn't exist yet -- `feature_flag_exposures` was created March 27, 2024 (1711569269846).

Reply:
uhhh good catch wtf. let me try that again